StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/SavedPaymentMethodFormFactory/SavedPaymentMethodFormFactory.swift

```diff
@@ -12,13 +12,14 @@ import Foundation
 import UIKit
 
 protocol SavedPaymentMethodFormFactoryDelegate: AnyObject {
-    func didUpdate(_: Element, didUpdateCardBrand: Bool)
+    func didUpdate(_: Element)
 }
 
 class SavedPaymentMethodFormFactory {
```

Comment:
This class seems fishy. It's called a "Factory" which implies it's responsibility is to make instances of something (a saved card form?), but it
- owns the form it creates
- is the delegate of the form it creates
- doesn't even have a public method to create instances of a saved card form?

At the minimum it seems like this should be renamed to some sort of "Controller".

Reply:
Created a task here: https://jira.corp.stripe.com/browse/MOBILESDK-3160

Reply:
I will handle this as part of this PR.